app/components/Form/ApplicationForm.tsx

```diff
@@ -320,7 +343,8 @@ const ApplicationForm: React.FC<Props> = ({
         areAllAcknowledgementsChecked &&
         jsonData?.review?.acknowledgeIncomplete &&
         !isSubmitted &&
-        isEditable
+        isEditable &&
+        !isProjectAreaOpen
```

Comment:
Should probably include in the dependency array - could cause false errors when attempting to submit

Reply:
yup, great catch!